src/EnergyPlus/DXCoils.cc

```diff
@@ -14292,7 +14292,7 @@ Label50: ;
 		}
 		return SHR;
 	}
-
+	
```

Comment:
You might see if Visual Studio has a setting for automatically deleting any trailing whitespace when saving a file.  @mjwitte do you know of this setting?